### PR TITLE
Move the index mapping definitions to standalone json files

### PR DESCRIPTION
**Pull request by pakio**
*Issue #, if available:* #138 

*Description of changes:*
Move the test index mapping definition, which directly written in code, to standalone json files.

Name of these json files are made referring to the function name.
I referred [this](https://github.com/opendistro-for-elasticsearch/sql/blob/master/src/test/java/com/amazon/opendistroforelasticsearch/sql/esdomain/mapping/FieldMappingsTest.java#L45) code to load each json file placed in `resources/mappings/**'

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/TestUtils.java [hunk 1]
```diff
@@ -127,566 +130,196 @@ public static Response performRequest(RestClient client, Request request) {
...
-        return "{  \"mappings\": {" +
-                " \"properties\": {\n" +
-                "          \"phrase\": {\n" +
-                "            \"type\": \"text\",\n" +
-                "            \"store\": true\n" +
-                "          }" +
-                "       }"+
-                "   }" +
-                "}";
+        String mappingFile = "mappings/phrase_index_mapping.json";
```
Comment (penghuo) [inline src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/TestUtils.java:143]: all the method have almost same implementation, the only different is mappingFile name, could you simply it?

**Comment by penghuo** [comment]
Thanks for the change.
Could you sumbit the PR against the develop branch? We plan to make master stable only for release purpose. All the new feature change will push to develop branch.

**Comment by pakio** [comment]
Sure, But I found a lot of differences between master branch and develop branch, and there are three different `TestUtils` (but same code) at develop branch.
Should I apply same changes to all file?

File: src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/TestUtils.java (hunk 1 above)
Comment (pakio) [reply src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/TestUtils.java]: Thank you for the review! I made a function to make logic and error handling common.
Also found `fileToString` method in TestUtils, so i stopped using Resources and used that method.

**Comment by chloe-zh** [comment]
> Sure, But I found a lot of differences between master branch and develop branch, and there are three different `TestUtils` (but same code) at develop branch.
> Should I apply same changes to all file?

Thanks for the changes! There should only one `TestUtils` in dev branch, it's located in `integ-test` module and in the directory `integ-test/src/test/../util`.

**Comment by chloe-zh** [comment]
> Sure, But I found a lot of differences between master branch and develop branch, and there are three different `TestUtils` (but same code) at develop branch.
> Should I apply same changes to all file?

Thanks for the changes! We are migrating the SQL codebase to a new architecture so there are still some pending changes, sorry to make you confused. We will clean up redundant codelines/files in the future.

For the index mapping methods, only `TestUtils` in integ-modul under `src/test/java/../legacy/` is called so it's fine to apply the changes only to this file.

But it looks like you are making changes in the one under `src/test/java/../util/` for now so it's actually skip the mapping methods you added. Could you move the changes toward `integ-test/src/test/java/../legacy/TestUtils` please?

**Comment by pakio** [comment]
Thanks for the detail information! Now I understood what should I do!
I've moved every my changes which was applied to `legacy/..` and index definition files into under `integ
-test/src/test/..`.
Could you please check if my change is alright or need more change?

**Comment by pakio** [comment]
@dai-chen Thanks for reviewing! I've already changed code at TestUtils.java to use these new json files. Is that enough change? or if it's not, I can make more change as needed !

**Comment by dai-chen** [comment]
> @dai-chen Thanks for reviewing! I've already changed code at TestUtils.java to use these new json files. Is that enough change? or if it's not, I can make more change as needed !

I may miss something but I only see the `TestUtils` class in legacy module is removed?

**Comment by chloe-zh** [comment]
> > @dai-chen Thanks for reviewing! I've already changed code at TestUtils.java to use these new json files. Is that enough change? or if it's not, I can make more change as needed !
> 
> 
> 
> I may miss something but I only see the `TestUtils` class in legacy module is removed?

Hi Chen @dai-chen , there are too many code lines removed in `TestUtils` so it doesn’t show up by default. I guess that’s what confused you?

**Comment by dai-chen** [comment]
> > > @dai-chen Thanks for reviewing! I've already changed code at TestUtils.java to use these new json files. Is that enough change? or if it's not, I can make more change as needed !
> > 
> > 
> > I may miss something but I only see the `TestUtils` class in legacy module is removed?
> 
> Hi Chen @dai-chen , there are too many code lines removed in `TestUtils` so it doesn’t show up by default. I guess that’s what confused you?

Yeah, I missed that and thought the whole file was removed :) Thanks!

**Pull request merged**